build.gradle

```diff
@@ -216,8 +208,14 @@ private def colorize(String color, String message) {
 }
 
 repositories {
-    mavenLocal()
+//    mavenLocal() // uncomment for testing with local maven
```

Comment:
why would you disable using the local repo? This would force developers to uncomment when developing locally where one needs to deploy to the local repo and consume in plugin that depends on intellij-kubernetes.

Reply:
I don't know why, but when I try to create a local intellij-common-ui-test-library and use it (mavenLocal () is uncommented), intellij-kubernetes won't build up. It works great with intellij-rsp, but not with kubernetes. This can be confusing, so I commented on this part, if someone uncomment it and project will not build, they will know why.

Reply:
would be important to find out why this happens. What's the error that you get?